Tools/GNUMake/sites/Make.nersc

```diff
@@ -24,6 +24,12 @@ ifeq ($(which_computer),$(filter $(which_computer),perlmutter))
   endif
   endif
 
+  ifeq (($USE_CUDA),TRUE)
+    includes += -Xcompiler="-march=znver3" 
```

Comment:
This will work.  But `includes` are for `-Ixxx`.

Reply:
Ah true. I'll move this to CXXFLAGS and CFLAGS when I get a chance.